packages/flutter/lib/src/widgets/image.dart

```diff
@@ -639,50 +933,70 @@ class _ImageState extends State<Image> {
   void _resolveImage() {
     final ImageStream newStream =
       widget.image.resolve(createLocalImageConfiguration(
-          context,
-          size: widget.width != null && widget.height != null ? Size(widget.width, widget.height) : null,
+        context,
+        size: widget.width != null && widget.height != null ? Size(widget.width, widget.height) : null,
       ));
     assert(newStream != null);
     _updateSourceStream(newStream);
   }
 
-  void _handleImageChanged(ImageInfo imageInfo, bool synchronousCall) {
+  ImageStreamListener get _listener => ImageStreamListener(
+    _handleImageFrame,
+    onChunk: widget.loadingBuilder == null ? null : _handleImageChunk,
+  );
```

Comment:
maybe this should be a method, so you can still use it in didUpdateWidget

Reply:
Done